src/parquet/util/schema-util.h

```diff
@@ -0,0 +1,84 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Thought of sticking them there (or schema.cc), but it seems to me that schema.h is long as it plus it should be more about parquet schema objects declaration, and whats in schema-util is just aux aux. functions which are more implementation details.